flashinfer/gemm.py

```diff
@@ -1550,12 +1547,12 @@ def bmm_fp8(
             dtype=dtype,
         )
 
+    workspace_buffer = _get_cache_buf(
```

Comment:
Why not separating the cache of different backends and allocate cudnn workspace inside `execute_cudnn_gemm_with_per_tensor_q_graph` after `graph.get_workspace_size()` is calculated.